app/code/Magento/Backend/view/adminhtml/templates/widget/tabshoriz.phtml

```diff
@@ -19,7 +19,7 @@ use Magento\Framework\View\Helper\SecureHtmlRenderer;
     <?php foreach ($tabs as $_tab): ?>
         <?php $tabId = $block->getTabId($_tab) ?>
         <?php $_tabClass = 'tab-item-link ' . $block->getTabClass($_tab) . ' ' .
-            (preg_match('/\s?ajax\s?/', $_tab->getClass()) ? 'notloaded' : '') ?>
+            (preg_match('/\s?ajax\s?/', $_tab->getClass() ?? '') ? 'notloaded' : '') ?>
```

Comment:
Should it be fixed in `\Magento\Backend\Block\Widget\Tabs::getTabClass` intead?

Reply:
@fascinosum  the problem is in a method 'getClass', and $_tab can be a DataObject